apps/ui/src/composables/useWalletConnect.ts

```diff
@@ -26,8 +26,8 @@ async function getConnector() {
       projectId: import.meta.env.VITE_WC_PROJECT_ID
     }),
     metadata: {
-      name: 'Snapshot X',
-      description: 'Snapshot X',
+      name: 'Snapshot',
+      description: 'Where decisions get made',
       url: 'https://snapshotx.org/',
```

Comment:
We don't use the new url `snapshot.box` here ?

Reply:
We should, good point, URL is was wrong here btw snapshotx.org